src/components/Tables/Table.tsx

```diff
@@ -3,9 +3,15 @@
 import React, { useState, useEffect } from "react";
 import DataTable, { TableColumn } from "react-data-table-component";
 import TableFilterBar, { FilterText } from "@/components/Tables/TableFilterBar";
+import { styled } from "styled-components";
+import { NoSsr } from "@mui/material";
+import SpeechBubbleIcon from "../Icons/SpeechBubbleIcon";
```

Comment:
@ syntax please, just makes moving files and navigating the repo etc a lot easier :)